pages/hackathon/[slug]/index.js

```diff
@@ -347,7 +347,7 @@ function Leaderboard({ slug, status, token }) {
 				console.error(err);
 			});
 		setLoading(false);
-	}, [token]);
+	}, [token, slug]);
```

Comment:
I think this change isn't needed, since the leaderboard component is always called when the hackathon details have been fetched. (And since that API requires the `slug`, the slug is obviously present and won't change for that page).

Reply:
it was a warning by react hooks plugin

Reply:
Oh, no problem then.